web/Views/Shared/_Layout.cshtml

```diff
@@ -23,17 +23,16 @@
                         <li class="nav-item">
                             <a class="nav-link text-dark" asp-area="" asp-controller="Home" asp-action="Index">Home</a>
                         </li>
-                        @if (User.Identity?.IsAuthenticated == true)
+                        @if (User.Identity?.IsAuthenticated == true && User.IsInRole("Admin"))

```

Comment:
Do we need the `User.Identity?.IsAuthenticated == true` check? I don't really mind keeping it for explicitness, but I would expect the `IsInRole` check to cover it.

Reply:
Yeah that's a very good point! I think I'll remove it for neatness :)